statements.py

```diff
@@ -0,0 +1,46 @@
+#input data
+current_bal: int = 500
+min_withdrawal: int = 20
+max_withdrawal: int = 1000
+
+#Exit function
+def exit():
+    print ("Thank you for choosing our bank! Have a good day!")
+def failure():
+    print ("Transaction failed. Try again")
+
+
+#Transaction function
+def transaction(current_bal):
```

Comment:
Don't forget a doc string for your function, it serves as a documentation for yourself and others